src/pybind/mgr/dashboard_v2/frontend/README.md

```diff
@@ -1,44 +1,48 @@
-# Openattic
+# Dashboard
 
 This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 1.6.3.
 
+## Installation
+
+Run `npm install` to install the required packages locally.
+
```

Comment:
You should mention here that if someone does not want to install the angular cli globally all commands that begin with `ng` need an additional `npm run` before it.